neofetch

```diff
@@ -2202,8 +2202,32 @@ get_cpu() {
                     [[ -z "$cpu" ]] && cpu="$(awk -F':' '/family/ {printf $2; exit}' "$cpu_file")"
                 ;;
 
+                "aarch64" | "armv7l")
+                    if grep -q "Raspberry Pi" /proc/device-tree/model; then
+                        cpu_revision="$(awk -F':' '/Revision/ {print substr($2,4,1); exit}' "$cpu_file")"
+                        case $cpu_revision in
+                            "0")
+                                cpu="BCM2835"
+                            ;;
+    
+                            "1")
+                                cpu="BCM2835"
+                            ;;
+    
+                            "2")
+                                cpu="BCM2837"
+                            ;;
+    
+                            "3")
+                                cpu="BCM2711"
+                            ;;
+                        esac
+
+                    fi
+                ;;&
+
```

Comment:
This should be fixed by #2139, it uses a more generic method to lookup ARM cpu names (by using `lscpu`).
Can you verify if #2139 works on raspi?

Reply:
Working on a Raspberry Pi 4B.

![image](https://user-images.githubusercontent.com/1096701/182050980-8482ccc5-3809-4a6e-9058-9871aa0c650b.png)

Reply:
> Working on a Raspberry Pi 4B.
> 
> ![image](https://user-images.githubusercontent.com/1096701/182050980-8482ccc5-3809-4a6e-9058-9871aa0c650b.png)

Great! Then we don't need this manual CPU lookup